src/index.ts

```diff
@@ -1,19 +1,15 @@
-import { Server } from 'http';
-import * as path from 'path';
-
-import * as _express from 'express';
-import * as _ from 'lodash';
```

Comment:
These are also type-only imports that made sense to be separate (and I expect there'll probably be a bunch more so I'll stop commenting on them now)